### DISCUSSION TITLE
Does KVS still work on 126?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1130] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Does KVS still work on 126?

**Comment by s0urce-c0de** [comment]
i don't think so...

**Reply by Balistic123** [reply]
why wouldn't it? would it work on 120?

**Reply by s0urce-c0de** [reply]
i have no idea, but i tried on v124 and it didn't work, so google won't do regression.

**Reply by Balistic123** [reply]
so I have a kernel version of 3 so that means I can downgrade from current version to 120 correct, I wonder if it works on 120?

**Reply by s0urce-c0de** [reply]
try it.

**Reply by joshuajohncohen** [reply, marked as answer]
I can confirm KVS does not work on 120 (kernver 3) at least for the octopus board

Also, from KVS creator (kxtz): "Google's Cr50/Ti50 TPM2 implementation also stops writing to the 0x1008 index if FWMP is enabled, unfortunately."

So if FWMP (aka block devmode) is enabled KVS is a lost cause pretty much
